internal/resource/pr_automation_trigger.go

```diff
@@ -0,0 +1,115 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"terraform-provider-plural/internal/client"
+	"terraform-provider-plural/internal/common"
+	"terraform-provider-plural/internal/model"
+)
+
+var _ resource.ResourceWithConfigure = &prAutomationTriggerResource{}
+
+func NewPrAutomationTriggerResource() resource.Resource {
+	return &prAutomationTriggerResource{}
+}
+
+type prAutomationTriggerResource struct {
+	client *client.Client
+}
+
+func (in *prAutomationTriggerResource) Metadata(_ context.Context, request resource.MetadataRequest, response *resource.MetadataResponse) {
+	response.TypeName = request.ProviderTypeName + "_pr_automation_trigger"
+}
+
+func (in *prAutomationTriggerResource) Schema(_ context.Context, _ resource.SchemaRequest, response *resource.SchemaResponse) {
+	response.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"pr_automation_id": schema.StringAttribute{
+				Description:         "ID of the PR Automation that should be triggered.",
+				MarkdownDescription: "ID of the PR Automation that should be triggered.",
+				Required:            true,
+				Validators: []validator.String{
+					stringvalidator.LengthAtLeast(1),
+				},
+			},
+			"repo_slug": schema.StringAttribute{
+				Description:         "Repo slug of the repository PR Automation should be triggered against. If not provided PR Automation repo will be used. Format: <userOrOrg>/<repoName>",
```

Comment:
the example in the docstring should make it explicit that's the format for github (it's actually different for Gitlab, but not bitbucket), something like:

`Example format for a github repository: <user-or-org-name>/<repo-name>`

Reply:
done